packages/blade/.babelrc.js

```diff
@@ -38,6 +39,7 @@ const configs = {
             runtime: 'automatic',
           },
         ],
+        'react-native-reanimated/plugin',
```

Comment:
how did this even worked in development then? 😮

Reply:
Didn't run motion on native since there were no stories 🤦‍♂️